### DISCUSSION TITLE
discussion...

### DISCUSSION THREAD

**Discussion by tsitsvero** [tsitsvero/hodgelaplacians#3] [Show and tell]
new question

**Comment by DanilTap** [comment]
How did you create this discussion?

**Reply by tsitsvero** [reply]
In Settings check "Discussions" and Discussions tab will appear

**Comment by Narayan-030** [comment]
I was wondering how to extract the simplices corresponding to the indexes of the say L2 Laplacian(Like what triangles do the entries in the matrix correspond to)

**Reply by Narayan-030** [reply]
Also just wanted to confirm that the order of the simplices that I feed in won't matter right?

**Reply by tsitsvero** [reply]
The order of vertices when defining simplices matters since it defines orientation.
